.github/workflows/accept-promotion-to-master.yaml

```diff
@@ -0,0 +1,22 @@
+name: Accept promotion PRs to master
+
+on:
+  check_suite:
+    types: 
+      - completed
+
+jobs:
+  # Only auto-approve PRs to master if they came from development in this fork
+  accept-promotion-to-master:
+    if: github.head_ref == 'development' && github.base_ref == 'master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: accept
+        uses: pascalgn/automerge-action@v0.6.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.MERGE_TOKEN }}
+          MERGE_LABELS: automerge,!block merge
+          MERGE_METHOD: rebase
```

Comment:
The reason I went with merge commits is because if you use `squash` or `rebase` on merging from development to master, you end up with completely divergent histories. This means that the PR commit list grows increasingly large over time. If you instead use `merge`, you retain the commit IDs master's history, and each promotion PR commit list is just the commits that have been added to development since the last promotion.

Reply:
Yes, you are right.